### ISSUE TITLE
Content recommendations based on recent history

### ISSUE THREAD

**Issue by sleetx** [stashapp/stash#3074]
Here's an idea based off a feature on a lot of tube sites: providing a page with "recommended" content from your library based on recent viewing history. This idea is different from the current filter system in the sense that it's adjusting its parameters automatically based on your recent history.

I'd imagine for Stash, the logic for serving up your library content would be primarily tag-based, with the most recently viewed tags weighted more heavily (particularly if it appears multiple times). The algorithm could also weight content by specific performers or studios if a certain one is more prominent in your history.

This feature is suggested primarily for scenes, although I suppose it could also work for galleries, images, performers, or movies depending on their tagged status.

**Comment by skier233** [comment]
I agree for the need for this. I'm working on an AI for content rating/suggestion. Also, once the rating system PR goes in that could be used for this as well. I'm not currently sure the best way this would be done but it would definitely be useful.
